WebRoot/messages/ZaMsg.properties

```
@@ -1183,6 +1183,7 @@ LBL_zimbraPrefCalendarShowPastDueReminders = Enable past due reminders
 MSG_zimbraPrefCalendarShowPastDueReminders = Enable past due reminders
 LBL_zimbraPrefAppleIcalDelegationEnabled = Use iCal delegation model for shared calendars for CalDAV interface
 MSG_zimbraPrefAppleIcalDelegationEnabled = Use iCal delegation model for shared calendars for CalDAV interface
+LBL_zimbraPrefMailDeliveryStatusNotification = Enable delivery status
```

Is there any reason why the message does not include "notification"?
```suggestion
LBL_zimbraPrefMailDeliveryStatusNotification = Enable delivery status notification
```

---

I have removed both and as per Lave's suggestion adding LBL_zimbraDeliveryStatusNotificationEnabled = Delivery status notification

---

Thank you for the update.